lib/rack/tracker/google_analytics/google_analytics.rb

```diff
@@ -28,6 +28,19 @@ def tracker
     options[:tracker].respond_to?(:call) ? options[:tracker].call(env) : options[:tracker]
   end
 
+  def tracker_options
+    @tracker_options ||= begin
+      tracker_options = {}
+
+      tracker_options[:cookieDomain] = options[:cookie_domain] if options[:cookie_domain]
+
+      user_id = options[:user_id].call(env) if options[:user_id]
+      tracker_options[:userId] = "#{user_id}" if user_id.present?
+
+      tracker_options
+    end
+  end
```

Comment:
good point and useful for the cookie_domain with an i18n multihost setup 17000d348411e38d989e90e7bc705d22f4b05dad